packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -224,6 +224,13 @@ export const selectDevice = (device?: Device | TrezorDevice) => (
     });
 };
 
+/**
+ * Toggles remembering the given device. I.e. if given device is not remembered it will become remembered
+ * and if it is remembered it will be forgotten.
+ * @param forceRemember can be set to `true` to remember given device regardless if its current state.
```

Comment:
I run into this action over the weekend and I just gave up. Having rememberDevice action that actually causes forgetting the device was really a mistake from the start. Maybe it should be renamed to `toggleRememberDevice` or something (and `rememberDevice` could be separate action that truly only remember device. I am not sure what was the usecase anymore and why we couldn't just use 2 separate actions...

Anyway just an ideas, big thank you for taking the time and documenting it!

Reply:
> Having rememberDevice action that actually causes forgetting the device was really a mistake from the start.

I guess what happened is that the method used to fit the UI layer (there's toggle that always switches from one state to the other or vice versa) but then later was used on other places too and extended with the `force` parameter which made it a real mess. .)

I wanted to refactor it to the two simple methods but I just wan't confident enough without the help of static type checking (the actions and their reducers are not 100% traceable by the TS compiler.) I can try to do the refactor (within this PR) if you promise to give it a throughout review. Deal?

Reply:
If we want to do bigger refactor (and I am all for it) then I would prefer to merge this after the freeze, there are already too many things that could break 😁